scripts/emulator-tests/functionsEmulatorRuntime.spec.ts

```diff
@@ -297,7 +297,8 @@ describe("FunctionsEmulator-Runtime", function () {
         expect(runtime.sysMsg["non-default-admin-app-used"]?.length).to.gte(1);
       });
 
-      it("should route all sub-fields accordingly", async () => {
+      // TODO(taeold): Fix and restore this test.
+      it.skip("should route all sub-fields accordingly", async () => {
```

Comment:
To fix this test:


```diff
- JSON.stringify(require("firebase-admin").firestore.FieldValue.increment(4))
+ JSON.stringify(require("firebase-admin/firestore").FieldValue.increment(4))
```

IIUC, Admin SDK also went through a large refactor in v10 to be modular, and we need to update the import syntax here...